data/changelog.txt

```diff
@@ -62,6 +72,9 @@ Fixes mobile home park road connections
 Flush map buffers after failing to create starting location
 Smart controller supports using only one engine
 Disassembly doesn't return items with UNRECOVERABLE flag
+Fix diary text navigation wrapping
+Fix display of diagonal vehicle glyphs
+Eliminate unnecessary long road connections
```

Comment:
```suggestion
```

Reply:
I think changing this might deserve to be in the changelog, but I also couldn't really tell how impactful it was.